### DISCUSSION TITLE
My github page will redirect to another domain I removed.

### DISCUSSION THREAD

**Discussion by Thecartoondude** [community/community#67249] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello! I am making a rep called PBS-Kids-2006 but the site still redirects to the custom domain I removed.
https://thecartoondude.github.io/PBS-Kids-2006/
Can you fix the problem?

**Comment by queenofcorgis** [comment, marked as answer]
Have you tried clearing your cache? It sounds simple but that may be the issue.

**Reply by Thecartoondude** [reply]
I did and it worked! Thank you!

**Comment by fanglong-it** [comment]
Can you describe more detail about the project ?, we can't help if doesn't understand what the issue

**Reply by Thecartoondude** [reply]
I made a custom domain but then I decided to remove it. But it still redirects to the domain I removed. It works fine on other browsers. But not chrome at all.

**Comment by queenofcorgis** [comment]
Glad we could get this solved! I reopened this for better discoverability should other users encounter the same 😄
